pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -456,6 +463,11 @@ func (a *Bootstrap) Load(f asset.FileFetcher) (found bool, err error) {
 		return false, errors.Wrapf(err, "failed to unmarshal %s", bootstrapIgnFilename)
 	}
 
+	report := validate.ValidateWithoutSource(reflect.ValueOf(config))
```

Comment:
would it be possible to add a small unit-test for the load function..

the Generate one might be too much overhead for now and can be skipped, but Load unit-test should be easy...?

Reply:
ok, sure